lang/ilias_fr.lang

```diff
@@ -1945,7 +1945,6 @@ auth#:#login_page_switch_rte#:#Utiliser l'Editeur de Texte Riche
 auth#:#lti_consumer_inactive#:#Le consommateur d’outils LTI est désactivé.
 auth#:#saml_tab_head_idp#:#IDP
 awrn#:#awareness_now_online#:#Maintenant en ligne
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.